poe-api/src/poe.py

```diff
@@ -147,6 +147,16 @@ def get_bot_names(self):
       bot_names[bot_nickname] = bot_obj["displayName"]
     return bot_names
   
+  def get_remaining_messages(self, chatbot):
+    url = f'https://poe.com/_next/data/{self.next_data["buildId"]}/{chatbot}.json'
+    limited_chatbots = ["GPT-4", "Claude+"]
+    if chatbot in limited_chatbots:
```

Comment:
There's no need to make the request manually when there's already a `client.get_bot` function for this.